src/libclipper/src/rpc_service.cpp

```diff
@@ -28,16 +29,19 @@ namespace clipper {
 namespace rpc {
 
 constexpr int INITIAL_REPLICA_ID_SIZE = 100;
+constexpr long CONTAINER_ACTIVITY_TIMEOUT = 30000;
```

Comment:
Nit: Make this `CONTAINER_ACTIVITY_TIMEOUT_MILLIS`. Specifying units of time in these variable names makes it easier to understand their meaning.